common/actions/findUsers.js

```diff
@@ -0,0 +1,50 @@
+import {getGraphQLFetcher} from 'src/common/util'
+
+export const FIND_USERS_REQUEST = 'FIND_USERS_REQUEST'
+export const FIND_USERS_SUCCESS = 'FIND_USERS_SUCCESS'
+export const FIND_USERS_FAILURE = 'FIND_USERS_FAILURE'
+
+function findUsersRequest() {
+  return {type: FIND_USERS_REQUEST}
+}
+
+function findUsersSuccess(users) {
+  return {type: FIND_USERS_SUCCESS, users}
+}
+
+function findUsersFailure(error) {
+  return {type: FIND_USERS_FAILURE, error}
+}
```

Comment:
These functions are super simple and unnecessary; let's just return these constants directly in the response handling below.